packages/components/Checkbox/src/Checkbox.settings.ts

```diff
@@ -28,9 +28,10 @@ export const settings: IComposeSettings<ICheckboxType> = [
         height: 14,
         width: 14,
         marginRight: 4,
+        marginLeft: 4,
```

Comment:
One quick note, I like the idea of using things like `marginStart` and `marginEnd`. Those will do the right thing in RTL and LTR I believe and depending on exactly what you want you may not need `marginEnd`

Reply:
This is exactly what I was looking for, thank you! I'm updating it now.